bin/test-workflow/policy/load_policies.sh

```diff
@@ -80,6 +83,12 @@ for app_name in "${APPS[@]}"; do
   conjur variable values add "my-app-db/prod/username" "prod-env-username"
   conjur variable values add "my-app-db/prod/port"     "12345"
   conjur variable values add "my-app-db/prod/cert-base64" "LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSURwRENDQW95Z0F3SUJBZ0lRRmZuTzFld2c1bUNoUi8rUlRWeDJQekFOQmdrcWhraUc5dzBCQVFzRkFEQVkKTVJZd0ZBWURWUVFERXcxamIyNXFkWEl0YjNOekxXTmhNQjRYRFRJeE1USXdOekUzTlRneE9Wb1hEVEl5TVRJdwpOekUzTlRneE9Wb3dHekVaTUJjR0ExVUVBeE1RWTI5dWFuVnlMbTE1YjNKbkxtTnZiVENDQVNJd0RRWUpLb1pJCmh2Y05BUUVCQlFBRGdnRVBBRENDQVFvQ2dnRUJBTHZ1SEIwMWo2YytnVVBTaW5Ub2czQ09FeWUrd1RxZ2VPYlgKK1YwSnNlQnhSTnpHTWlSeWljWXIxWG9MeWh1MXpYcHEzQ1JKbDdZaDc1TGtQWENCblo3UlBxYW1yZXFxMWYyZwp2ODJidnNxRkhKQ3Y2WlhLVlNTRTJQY2xEMDZOZjFCUzVoc1FhTjhlblJCRURnTzRqbUszUjBUajBuWGNXMjJBCmZjSlVxQ3MvZm9GVHNVOWxmYitLNDVSWlBGWjRJNnpCQ25QcHg3bzJXVjNPUnpqQ0s5M1lDb2U3OHVVWmNFaGwKK2ZIdGtnQ3N2T1pFa2EvYjRpUjJZRUJmNWlxLzdjSjU4OFpEdVZwL0FyWW9tNHB0dnc0R2srbEdxRytaSjg5eQo1UElBaEplS3NvS3N6Y0xyR0tvYndQdERMY3doYzhQaTJNS2FMZTJsUUhRU1J5cDF1djhDQXdFQUFhT0I1akNCCjR6QU9CZ05WSFE4QkFmOEVCQU1DQmFBd0hRWURWUjBsQkJZd0ZBWUlLd1lCQlFVSEF3RUdDQ3NHQVFVRkJ3TUMKTUF3R0ExVWRFd0VCL3dRQ01BQXdId1lEVlIwakJCZ3dGb0FVN3J3RmVJV0lhbmNtQkdoMGFjUjZPY0pBTHN3dwpnWUlHQTFVZEVRUjdNSG1DRUdOdmJtcDFjaTV0ZVc5eVp5NWpiMjJDQ21OdmJtcDFjaTF2YzNPQ0ZXTnZibXAxCmNpMXZjM011WTI5dWFuVnlMVzl6YzRJWlkyOXVhblZ5TFc5emN5NWpiMjVxZFhJdGIzTnpMbk4yWTRJblkyOXUKYW5WeUxXOXpjeTVqYjI1cWRYSXRiM056TG5OMll5NWpiSFZ6ZEdWeUxteHZZMkZzTUEwR0NTcUdTSWIzRFFFQgpDd1VBQTRJQkFRQzRjZEtWMGZ4NC96WkZwYUx1bmM0MGY0blkwUHZzRFRaZUNCM0w1M1c5M29hVGFjeTFEcWVFCnVScWxVT05yWWNsQitzOUYyTWVmRHBTK0swTHlVV21hVG9oM05JWGtKTDZHNS9TTHVuN01ZRXJwMXN0YVpDY3UKMkxQcjR1bDJMRmY3UVJjRytwRXIvTmxmay9RaENNb1NCYk1MNmZvYWtSQzNpTzA2bEUrMWk2MHBSdzdkdW90YgpoV3R0cUhqSTFMc1QxZ09neDhOVTNLeUczNERqNURGZFdQbU1FckdjWlNpMXNjdmlLL0UvMTkyUUxGbnB4UlR2CnltQ1J0eWpaUjFyaUM5ZkhqcUhuNGZ1bVV3eUJ5aEJpUmVmNXhGVHBiL2lQZzB6a0lIeENzd3ZQY3lod1djaHMKZGlYeElPUEVxWGNNN0xRbnNvalBoUEt0VXlJRytKdWwKLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQoK"
+
+  echo "Adding JWT variables"
+  conjur variable values add conjur/authn-jwt/$AUTHENTICATOR_ID/jwks-uri $JWKS_URI
+  conjur variable values add conjur/authn-jwt/$AUTHENTICATOR_ID/token-app-property "sub"
+  conjur variable values add conjur/authn-jwt/$AUTHENTICATOR_ID/issuer "https://kubernetes.default.svc.cluster.local"
```

Comment:
It will be better to fetch issuer from OpenID configuration exactly like you're doing it for JWKS_URI
```
ISSUER=$($cli get --raw /.well-known/openid-configuration | jq '. issuer')
```

Reply:
Good point fixed. Thanks